.env.dist

```diff
@@ -0,0 +1,56 @@
+APP_NAME=Laravel
+APP_ENV=local
+APP_KEY=base64:3R7Tumc/H0dz8qwMa0rGdxoSDGOFZgcT3iKmEkDJQ8E=
+APP_DEBUG=true
+APP_URL=http://meetup
+APP_TIMEZONE='Europe/Berlin'
+
+LOG_CHANNEL=stack
+LOG_LEVEL=debug
+
+DB_CONNECTION=pgsql
+DB_HOST=db
+DB_PORT=5432
+DB_DATABASE=meetup
+DB_USERNAME=meetup
+DB_PASSWORD=password
+
+BROADCAST_DRIVER=log
+CACHE_DRIVER=file
+FILESYSTEM_DRIVER=local
+QUEUE_CONNECTION=redis
+SESSION_DRIVER=file
+SESSION_LIFETIME=120
+
+MEMCACHED_HOST=memcached
+
+REDIS_HOST=redis
+REDIS_PASSWORD=
+REDIS_PORT=6379
+REDIS_CLIENT=predis
+
+MAIL_MAILER=smtp
+MAIL_HOST=mailhog
+MAIL_PORT=1025
+MAIL_USERNAME=null
+MAIL_PASSWORD=null
+MAIL_ENCRYPTION=null
+MAIL_FROM_ADDRESS=null
+MAIL_FROM_NAME="${APP_NAME}"
+
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_DEFAULT_REGION=us-east-1
+AWS_BUCKET=
+AWS_USE_PATH_STYLE_ENDPOINT=false
+
+PUSHER_APP_ID=
+PUSHER_APP_KEY=
+PUSHER_APP_SECRET=
+PUSHER_APP_CLUSTER=mt1
```

Comment:
Do we need AWS and PUSHER variables in `.env` file?
We will use that in project?